backend/src/main/java/de/neuefische/teddyscoronadiaries/model/weather/WeatherCategories.java

```diff
@@ -0,0 +1,25 @@
+package de.neuefische.teddyscoronadiaries.model.weather;
+
+public enum WeatherCategories {
+    SN("Schnee"),
+    SL("Schneeregen"),
+    H("Hagel"),
+    T("Gewitter"),
+    HR("Starkregen"),
+    LR("Leichter Regen"),
+    S("Regen"),
+    HC("Bewölkt"),
+    LC("Leicht bewölkt"),
+    C("Sonnig");
+
+    public final String name;
+
+    private WeatherCategories(String name) {this.name = name;}
+
+    public static String getWeatherCategory(String category) {
+        for(WeatherCategories weatherCategory : values()) {
+            if(category.equals(weatherCategory.toString())) {return weatherCategory.name;}
+        }
+        return null;
```

Comment:
Small suggestion: Switch to an `Optional<T>` return type...
https://www.baeldung.com/java-avoid-null-check